packages/paste-core/utilities/flex/src/index.tsx

```diff
@@ -237,6 +240,181 @@ const Flex: React.FC<FlexProps> = ({
 };
 
 Flex.propTypes = {
+  as: PropTypes.oneOf([
```

Comment:
Can we make these reusable somehow? So they can be used across `Flex`, `Grid`, and `Column`. Is that over-optimizing?

Reply:
Yes, but I'm not sure where. And I don't _think_ I want to create another package right now...

Reply:
This should just be a string imo

Reply:
makes it so much easier if it's a string